src/index.css

```diff
@@ -90,6 +92,27 @@ body {
   }
 }
 
+@media screen and (min-width: 1024px) {
+  /* take only 1, 4, 7 + before-last */
+  .last-row-modifier:nth-child(3n - 2):nth-last-child(2),
+/* take 2, 3, 4, exclude 1, 4, 7, exclude dividable by 3 + take only last */
+/* = take 2, 5, 8 only last of those */
+.last-row-modifier:nth-child(n + 2):not(:nth-child(3n
+      -
+      2)):not(:nth-child(3n)):last-child {
+    grid-column: span 3 / span 3;
+  }
+
+  .last-row-modifier:nth-child(3n - 2):last-child {
+    grid-column: var(--col-span-full);
+  }
+}
+@media screen and (min-width: 450px) and (max-width: 1024px) {
+  .last-row-modifier:nth-child(odd):last-child {
```

Comment:
This can be done in `tailwind`, but I don't want to separate same logic for different screens

Reply:
Basically all of this can be done using tailwind, except `media 450-1024`, I guess

Reply:
Okay, okay, let's move it into tailwind :)